### PR TITLE
[update][Yu & Qing]return event in modal

### PR DESCRIPTION
**Pull request by chenqing2016**
return event object in modal box

### CODE REVIEW

File: src/components/button/Button.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ const styles = require("./button.scss");
 type ButtonType = "submit" | "reset" | "button";
 
 export interface IButtonProps {
-  onClick: () => any;
+  onClick: (event?: any) => any;
```
Comment (LingweiWang) [inline src/components/button/Button.tsx]: There should a event type for this

File: src/components/modalContent/ModalContent.tsx [hunk 2]
```diff
@@ -28,7 +28,9 @@ export class ModalContent extends React.Component<IModalContentProps, {}> {
             <Button
               className={styles.modalButton}
               label={this.props.leftButtonLabel}
-              onClick={this.props.leftButtonOnClick}
+              onClick={(event: any) => {
```
Comment (LingweiWang) [inline src/components/modalContent/ModalContent.tsx]:
```suggestion
              onClick={this.props.leftButtonOnClick}
```

File: src/components/modalContent/ModalContent.tsx [hunk 3]
```diff
@@ -37,7 +39,9 @@ export class ModalContent extends React.Component<IModalContentProps, {}> {
             <Button
               className={styles.modalButton}
               label={this.props.rightButtonLabel}
-              onClick={this.props.rightButtonOnClick}
+              onClick={(event: any) => {
```
Comment (LingweiWang) [inline src/components/modalContent/ModalContent.tsx]:
```suggestion
              onClick={this.props.rightButtonOnClick}
```

File: src/components/button/Button.tsx (hunk 1 above)
Comment (chenqing2016) [reply src/components/button/Button.tsx]: I already changed

**Pull request merged**